x-pack/plugins/security_solution/public/management/components/paginated_content/paginated_content.tsx

```diff
@@ -164,9 +164,16 @@ export const PaginatedContent = memo(
 
     const handleItemsPerPageChange: EuiTablePaginationProps['onChangeItemsPerPage'] = useCallback(
       (pageSize) => {
-        onChange({ pageSize, pageIndex: pagination?.pageIndex || 0 });
+        if (pagination?.pageIndex) {
+          onChange({
+            pageSize,
+            pageIndex: Math.floor((pagination?.pageIndex * pagination?.pageSize) / pageSize),
```

Comment:
because `pagination` is an optional param, you need to assign defaults to `pageIndex` and `pageSize` so that `undefined` is handled.
In addition, this code here does not know what the numbers for `pageIndex` and `pageSize` will be, it should also handle `NaN`. I'm thinking something like this might work:

```
const pageIndex: Math.floor((pagination?.pageIndex ?? 0 * pagination?.pageSize ?? 10) / pageSize);
```

Then here:

```suggestion
            pageIndex: isNaN(pageIndex) ? 0 : pageIndex
```